lupa/_lupa.pyx

```diff
@@ -92,6 +96,18 @@ cdef int _LUA_VERSION = lua.read_lua_version(NULL)
 LUA_VERSION = (_LUA_VERSION // 100, _LUA_VERSION % 100)
 
 
+if lua.LUA_VERSION_NUM >= 503:
+    LUA_MININTEGER, LUA_MAXINTEGER = (lua.LUA_MININTEGER, lua.LUA_MAXINTEGER)
+elif sizeof(lua.lua_Integer) >= sizeof(long long):  # probably not larger
+    LUA_MININTEGER, LUA_MAXINTEGER = (PY_LLONG_MIN, PY_LLONG_MAX)
+elif sizeof(lua.lua_Integer) >= sizeof(long):
+    LUA_MININTEGER, LUA_MAXINTEGER = (LONG_MIN, LONG_MAX)
+elif sizeof(lua.lua_Integer) >= sizeof(int):
+    LUA_MININTEGER, LUA_MAXINTEGER = (INT_MIN, INT_MAX)
+else:  # probably not smaller
```

Comment:
What about `short`? I mean, I wouldn't expect anyone to define `lua_Integer` as anything smaller than `long`, let alone `int`, but if we call out `char`, then I'd also expect `short int` on that list. (Mind you, I can live without it – I just find the jump from `int` to `char` striking.)

Reply:
You're right.